core/types/transaction.go

```diff
@@ -172,6 +178,8 @@ func (tx *Transaction) UnmarshalBinary(b []byte) error {
 	return nil
 }
 
+var weiPerGwei = new(big.Int).Exp(big.NewInt(10), big.NewInt(9), nil)
```

Comment:
Some unit denomination constants can be found here: https://github.com/ethereum/go-ethereum/blob/d8ff53dfb8a516f47db37dbc7fd7ad18a1e8a125/params/denomination.go#L20-L26

Reply:
ah beautiful! thank you :)